doc/users/explain/fonts.rst

```diff
@@ -60,50 +63,37 @@ Other font specifications which Matplotlib supports:
-those formats (for example, PDF, PS/EPS, SVG) allow embedding font data in such
-a way that when these documents are visually scaled, the text does not appear
-pixelated.
-
-This can be achieved by embedding the *whole* font file within the
-output document. However, this can lead to very large documents, as some
-fonts (for instance, CJK - Chinese/Japanese/Korean fonts) can contain a large
-number of glyphs, and thus their embedded size can be quite huge.
-
-Font Subsetting can be used before generating documents, to embed only the
-*required* glyphs within the documents. Fonts can be considered as a collection
-of glyphs, so ultimately the goal is to find out *which* glyphs are required
-for a certain array of characters, and embed only those within the output.
-
-.. note::
-  The role of subsetter really shines when we encounter characters like **ä**
-  (composed by calling subprograms for **a** and **¨**); since the subsetter
-  has to find out *all* such subprograms being called by every glyph included
-  in the subset, this is a generally difficult problem!
-
-Luckily, Matplotlib uses a fork of an external dependency called
-`ttconv <https://github.com/sandflow/ttconv>`_, which helps in embedding and
-subsetting font data. (however, recent versions have moved away from ttconv to
-pure Python for certain types: for more details visit
-`these <https://github.com/matplotlib/matplotlib/pull/18370>`_, `links <https://github.com/matplotlib/matplotlib/pull/18181>`_)
-
-| *Type 1 fonts are still non-subsetted* through Matplotlib. (though one will encounter these mostly via *usetex*/*dviread* in PDF backend)
-| **Type 3 and Type 42 fonts are subsetted**, with a fair amount of exceptions and bugs for the latter.
-
-What to use?
-------------
-Practically, most fonts that are readily available on most operating systems or
-are readily available on the internet to download include *TrueType fonts* and
-its "extensions" such as MacOS-resource fork fonts and the newer OpenType
-fonts.
+Font Subsetting
+~~~~~~~~~~~~~~~
+
+PDF and postscript support embedded fonts in the output files allowing the
+display program to correctly render the text, independent of what fonts are
+installed on the viewer's computer, without the need to pre-rasterize the text.
+This ensures that if the output is zoomed or resized the text does not become
+pixelated.  However, embedding full fonts in the file can lead to large output
+files, particularly with fonts with many glyphs such as those that support CJK
+(Chinese/Japanese/Korean).
+
+The solution to this problem is to subset the fonts used in the document and
+only embed the glyphs actually used.  This gets both vector text and small
+files sizes.  Computing the subset of the font required and writing the new
+(reduced) font are both complex problem and thus Matplotlib relies on a
+vendored fork of `ttconv <https://github.com/sandflow/ttconv>`_ and `fontTools
+<https://fonttools.readthedocs.io/en/latest/>`__.
```

Comment:
I would swap the order of these, so it's clear that `ttconv` is the only vendored fork.